src/v2/api/teacher/data/UpdateDisciplineTeacherData.ts

```diff
@@ -0,0 +1,8 @@
+import { TeacherRole } from '@prisma/client';
+
+export class UpdateDisciplineTeacherData {
```

Comment:
What is the purpose of this Data class and where it is used. 99% it must be named DTO or deleted.